src/input-field.tsx

```diff
@@ -65,6 +65,7 @@ export interface InputProps
     rows?: number
     label?: React.ReactNode
     id?: string
+	placeholder? : string
```

Comment:
this is duplicating prop from `React.HTMLProps<HTMLInputElement>` and isn't needed (also indentation uses tabs I think?)